From e51a49b09f680122a9600e793ea8367f8c00da0d Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Mon, 28 Feb 2011 13:08:21 +0800
Subject: [PATCH 25/30] WRHV: Cavium: Disable perf_counters.c build

GOS can't support physical PMR since CP0 perf counters registers
can't be accessed. So, only timer mode is supported.

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 arch/mips/cavium-octeon/Makefile |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/arch/mips/cavium-octeon/Makefile b/arch/mips/cavium-octeon/Makefile
index 3b7aed8..672c7b7 100644
--- a/arch/mips/cavium-octeon/Makefile
+++ b/arch/mips/cavium-octeon/Makefile
@@ -11,8 +11,12 @@
 
 obj-y := cpu.o setup.o serial.o octeon-platform.o octeon-irq.o csrc-octeon.o
 obj-y += dma-octeon.o octeon_info.o
-obj-y += octeon-memcpy.o perf_counters.o
 obj-y += pci-console.o
+obj-y += octeon-memcpy.o
+ 
+ifeq ($(CONFIG_WRHV),n)
+obj-y           += perf_counters.o
+endif
 
 obj-$(CONFIG_MTD)                     += flash_setup.o
 obj-$(CONFIG_SMP)                     += smp.o
-- 
1.6.5.2

